docs/api-docs/provider-apis/tax/overview.mdx

```diff
@@ -62,23 +55,16 @@ When you [Establish a Tax Provider connection](#establishing-a-connection), incl
   You must liaise with the merchant to provide them with customized URLs. BigCommerce merely consumes the URL values the merchant enters and uses the provided string value for Tax Provider API requests.
 </Callout>
 
-### Sandbox tax provider configuration
-
-Sandbox tax provider configurations are nested within your production tax provider configuration, as such tax providers requiring a sandbox tax provider configuration should also provide details for their production tax provider configuration.
+### Test tax provider (optional)
```

Comment:
We've had feedback that the terminology we were using was often confused - so I've adjusted this to focus on the reason tax providers may want to implement a "test tax provider".

I'm not 100% sold on the language in this area in particular.

One specific call out is the language "tax provider" that we use throughout this documentation - sometimes this means a more _technical_ tax provider ("tax provider instance") and sometimes it means the team building the solution ("tax provider"). For larger organisations we'd call the later "tax partner". We have also used the terminology "tax service" to describe the "tax provider instance" in some places.

I'm not completely sold on any particular change in this area, but thought I'd highlight if someone else has some ideas or preferences.

Reply:
@theromulans I'm thinking of referring to tax provider organizations/partners with "you" (like what I did in the [Tax Profile section](https://github.com/bigcommerce/docs/pull/130/files#diff-6454eb7541f7642d69afa823f3e2b7f3812936c2afd4fdc54094178546abda3fR45)). This would make sense since tax providers organizations/partners/team are the people using the doc (and second-person voice is also a style choice that we use elsewhere in BC docs).

Then we could use "Tax Provider instance" to refer to the technical tax provider service, like what you mentioned.

Reply:
^I made suggestions in a couple of places to illustrate what I mean, but I can go through the rest of the doc to make this change

Reply:
Great, thanks for the guidance here @bc-andreadao!

I've made some further adjustments to try and remove the language confusion here and use the language you've suggested - in particular with regard to perspective: more "You..." and less "The tax provider" third person.